tensorflow/lite/micro/examples/magic_wand/sparkfun_edge/accelerometer_handler.cc

```diff
@@ -132,13 +139,13 @@ TfLiteStatus SetupAccelerometer(tflite::ErrorReporter* error_reporter) {
   }
 
   if (lis2dh12_fifo_mode_set(&dev_ctx, LIS2DH12_BYPASS_MODE)) {
-    TF_LITE_REPORT_ERROR(error_reporter, "Failed to clear FIFO buffer.");
-    return 0;
+    error_reporter->Report("Failed to clear FIFO buffer.");
```

Comment:
Here is the internal error : `TF_LITE_REPORT_ERROR should be used instead of error_reporter, so log strings can be removed to save space if needed. ` , @oclyke can you please check this ?

Reply:
I've replaced that usage with the proper macro style and manually tested the changes. Everything is A-ok on this end. Thanks for reviewing.